### DISCUSSION TITLE
RM13 and field exosuit armor probably require stats rework.

### DISCUSSION THREAD

**Discussion by Firestorm01X2** [cataclysmbn/Cataclysm-BN#2048] [General]
RM13 turned off protection:  Bash: 4, Cut: 10 
RM13 turned on protection: Bash: 10, Cut: 25
Field combat exoskeleton (and helmet) protection: Bash: 36. Cut: 36

But at the same time  
MBR vest with ceramic plates protection: Bash: 18, Cut: 45

It is quite surprising that pinnacle of technology has less protection than normal military armour. Even considering 100% converage for power armour.

So RM13 and field exosuit armor probably require stats rework.

**Comment by Firestorm01X2** [comment]
My previous attempt:
https://github.com/cataclysmbnteam/Cataclysm-BN/pull/368

**Comment by Coolthulhu** [comment]
RM13 has high coverage, high environment resistance and some extra bonuses on top. MBR vest is just a vest.
Because of how coverage works, MBR doesn't guarantee that you can't be shot in the vital organs.

Exoskeleton could be buffed by adding Kevlar as one of the materials, so that it can protect from firearms better. But its bash protection doesn't need to stay as high as it is - sci-fi or not, being smashed against a wall should not be easily reduced to 0 damage.

**Comment by chaosvolt** [comment]
This might be obsoleted by https://github.com/cataclysmbnteam/Cataclysm-BN/pull/1493, as it added carbide to the material list, giving it 45 ballistic protection when turned on.
